index.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-const uuid = require('uuid/v4')
+const { v4: uuid } = require('uuid');
```

Comment:
Just noticed they don't use `;` at the end. Please remove it for consistency 😄

Reply:
Done 😄